openblas/src/main/java/org/bytedeco/openblas/presets/openblas_full.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2016-2025 Samuel Audet
+ *
+ * Licensed either under the Apache License, Version 2.0, or (at your option)
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation (subject to the "Classpath" exception),
+ * either version 2, or any later version (collectively, the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *     http://www.gnu.org/licenses/
+ *     http://www.gnu.org/software/classpath/license.html
+ *
+ * or as provided in the LICENSE.txt file that accompanied this code.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bytedeco.openblas.presets;
+
+import java.util.List;
+import java.util.ListIterator;
+import org.bytedeco.javacpp.ClassProperties;
+import org.bytedeco.javacpp.annotation.NoException;
+import org.bytedeco.javacpp.annotation.Platform;
+import org.bytedeco.javacpp.annotation.Properties;
+import org.bytedeco.javacpp.presets.javacpp;
+import org.bytedeco.javacpp.tools.Info;
+import org.bytedeco.javacpp.tools.InfoMap;
+
+/**
+ *
+ * @author Samuel Audet
+ * @author Dragan Djuric
+ */
+@Properties(inherit = openblas.class, global = "org.bytedeco.openblas.global.openblas_full", value = {
+    @Platform(include = {"openblas_config.h", "cblas.h", "lapacke_config.h", "lapacke_mangling.h", "lapack.h", "lapacke.h", "lapacke_utils.h"})})
```

Comment:
If the include list is the same as openblas.class, we probably don't need to add it here. Can you try to remove this line?

Reply:
Yes, of course. Sorry for these sloppy edits, I'm doing this through GitHub editor :) Update's comming in a minute.

Reply:
Fixed.

Reply:
Sorry, I misread that you mean about java imports. I'll be back in a minute :)

Reply:
Changed.

Reply:
Yes, yes, I know... Sorry. I hope the next edit will be ok :)

Reply:
Maybe the previous actions should be cancelled before they finish...

Reply:
Now the build works, but looks suspiciously quick in comparison to the last time it worked. As I don't know where to look for the resulting artifacts, I'll have to wait for your judgment.

Reply:
Right, it doesn't work, because what inherit does is closer to a dependency declaration, so the build doesn't reparse the same files. Meh, the list is not long, so let's just leave it like you had it. Sorry

Reply:
I've re-included it. Fingers crossed :)